docs/installation_configuration_guide/DesignerConfiguration.md

```diff
@@ -598,19 +598,21 @@ Tabs (in main menu bar, such as Scenarios etc.) can be configured in the followi
 ```
 By default, only `Scenarios` tab is configured.
 
-| Parameter name     | Type                | Description                                                |
-| --------------     | ----                | -----------                                                |
-| id                 | string              | Unique identifier                                          |
-| title              | string              | Title appearing in UI                                      |
-| type               | IFrame/Local/Remote | Type of tab (see below for explanation)                    |
-| url                | string              | URL of the tab                                             |
-| requiredPermission | string              | Optional parameter, name of [Global Permission](#security) |
-                                       
+| Parameter name     | Type                         | Description                                                |
+| --------------     |------------------------------| -----------                                                |
+| id                 | string                       | Unique identifier                                          |
+| title              | string                       | Title appearing in UI                                      |
+| type               | IFrame/Local/Remote/Url | Type of tab (see below for explanation)                    |
+| url                | string                       | URL of the tab                                             |
+| requiredPermission | string                       | Optional parameter, name of [Global Permission](#security) |
+| target             | string                       | Optional parameter, html a [target param](https://www.w3schools.com/tags/att_a_target.asp) |
```

Comment:
remove target from here

Reply:
Gracias amigo! I thought, I had removed it.